maro/simulator/scenarios/vm_scheduling/physical_machine.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Set
+
+from maro.backends.frame import NodeAttribute, NodeBase, node
+
+
+@node("pms")
+class PhysicalMachine(NodeBase):
+    """Physical machine node definition in frame."""
+    # Initial parameters.
+    id = NodeAttribute("i")
+    cpu_cores_capacity = NodeAttribute("i2")
+    memory_capacity = NodeAttribute("i2")
```

Comment:
do we have a smaller date type?

Reply:
i2 is smallest data type at this version, can be changed to unsigned byte with new version